topics/introduction/metadata.yaml

```diff
@@ -11,20 +11,13 @@ material:
   -
     title: "Introduction"
     type: "introduction"
+    name: "introduction"
     slides: "yes"
     contributors:
-      -
-        name: "Bérénice Batut"
-        github_username: "bebatut"
-      -
-        name: "Saskia Hiltemann"
-        github_username: "shiltemann"
-      -
-        name: "Anne Pajon"
-        github_username: "pajanne"
-      -
-        name: "Nicola Soranzo"
-        github_username: "nsoranzo"
+      - bebatut
+      - shitlemann
```

Comment:
shiltemann

Reply:
pffft :p ..thanks so much for catching all of these!